### DISCUSSION TITLE
Allow use of suppression comments from other linters and formatters

### DISCUSSION THREAD

**Discussion by eye-wave** [biomejs/biome#4736] [Ideas]
What if biome could treat `// prettier-ignore` like `// biome-ignore format. ` for simpler migration process?

**Comment by dyc3** [comment]
We could have something similar to the `--suppress` arg that was added in #4008 that would change the prettier ignores into biome's ignores? Because ideally you wouldn't be formatting the same file with biome and prettier.

**Reply by ematipico** [reply]
I don't see enough value in that. Users can do a search and replace from their editors, and it will work **better** than a possible implementation of ours.

Plus, they could expect the same for eslint suppression comments, and we can't map off of them to our suppressions.

**Comment by eye-wave** [comment]
Initially i thought about this feature as a "universal suppressor" comment for formatters and linters, but that would require some w3c like standards to be in place
